settings.py

```diff
@@ -4,10 +4,22 @@
 LOGGING_FORMAT = "%(asctime)s|%(levelname)s: sdx-store: %(message)s"
 LOGGING_LEVEL = logging.getLevelName(os.getenv('LOGGING_LEVEL', 'DEBUG'))
 
-MONGODB_URL = os.getenv("MONGODB_URL", "mongodb://localhost:27017")
+DB_HOST = os.getenv('POSTGRES_HOST', '0.0.0.0')
```

Comment:
Waiting for https://github.com/ONSdigital/sdx-store/pull/37 to get merged and will merge those changes in